mobile-widgets/qmlmanager.cpp

```diff
@@ -100,6 +100,13 @@ void QMLManager::btHostModeChange(QBluetoothLocalDevice::HostMode state)
 }
 #endif
 
+void QMLManager::btRescan()
+{
+#if defined(BT_SUPPORT)
+	BTDiscovery::instance()->BTDiscoveryReDiscover();
+#endif
+}
+
```

Comment:
see my comment - can you look at my PR from two minutes ago and then not do the BT_SUPPORT ifdefs?
Or if you disagree with my decision, keep this and NAK my PR :-)

Reply:
We were overlapping. I am fine with both. Pick what you want.